README.md

```diff
@@ -0,0 +1,118 @@
+# simba-starter
+
+A [lion](https://lion-web.netlify.app/) based design system as a starter kit
+for [Backlight](https://backlight.dev/), an all-in-one Design System platform,
+design inspired by [TailwindCSS](https://tailwindcss.com/).
+
+## [🔗 See it in action](https://backlight.dev/preview/5vtJtbY04aoD1dGKcsu1)
+
+Click the "Edit" button to also see the source code in the Backlight editor.
+
+## Features
+
+- 🍴 Easy to fork and create your own variation using [Backlight](https://backlight.dev/)
+- 📦 Available on NPM to extend or use directly
+- 🌙 Darkmode out of the box
+- 🦄 Theming capabilities, choose any color from Tailwind palette out of the box, with room for more!
+
+## Usage
```

Comment:
This usage goes with the npm section below.
We should start with "... Duplicate in Backlight... Publish to npm... and use it your app". Don't spend too much time explaining the  mechanism here. The product and the doc should do the rest. It will be easier to maintain the doc and the onboarding  than every single starter kit

Reply:
I think I changed it the way you expect 👍